gravitl/netclient
# Gra 1069/proxy multiserver peer deletion

testing steps:
-> enable proxy on the hosts
-> join two servers from two hosts and leave one of the host from one server and check peer connections if they are working
